### ISSUE TITLE
The grid and other components overlap each other when using view. 

### ISSUE THREAD

**Issue by ZijianZhang991203** [SHISME/react-native-draggable-grid#53]
The grid and other components overlap each other when using view, but works perfect when using scrollview as a wrapper. Is this a normal behaviour? Thanks!

**Comment by Ruzen1** [comment]
I think flex: 1 should do work. pass it to the view

**Comment by ZijianZhang991203** [comment]
> I think flex: 1 should do work. pass it to the view

Yeah. That works, but then when it has several lines and dynamically changed. It will overlap the stuff beblow.

**Comment by Ruzen1** [comment]
I think its the problem with the key I guess.
